src/Aliases.jl

```diff
@@ -8,6 +8,9 @@ include(joinpath(pathof(AbstractAlgebra), "..", "Aliases.jl"))
 @alias is_less isless
 @alias is_real isreal
 
+@alias eigenvalues eigvals
```

Comment:
AbstractAlgebra doesn't use `@alias` here (see https://github.com/Nemocas/AbstractAlgebra.jl/blob/53d640dc11871537becc8805519b3bdab3fe68c3/src/AbstractAlgebra.jl#L324-L333). I think we should do it here the same.
And in any case, I think this does not belong to `Aliases.jl` as we do not create an Oscar-internal alias here but this is more like an "import with a different name". And this needs to move earlier in the inclusion-chain so that we can already use the new name inside Nemo (e.g. your implementation).

Reply:
I didn't know there is more than one established way to "alias" something in our packages. I changed it around. Although now when one does `?eigvals`, one does not see the Nemo doc strings...